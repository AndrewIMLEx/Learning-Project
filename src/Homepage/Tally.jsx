import React from "react"

class Tally extends React.Component{
    render(){
        return(
            <>
            <div className="tallycard">
            <h4 style={{marginLeft:"2.8em",paddingBottom:"2em"}}>Game Tally</h4>
            <form id = "LevelForm">
            <span><label id = "won-label" htmlFor="won">Games Won</label>
            <label id = "Tally"/></span>
            &nbsp;&nbsp;
            <span><label id = "won-label" htmlFor="lost">Games Lost</label>
            &nbsp;<label id = "Tally"/></span>
             </form>
             <form id = "Gamesplayed">
             <label id ="played" htmlFor="gamesplayed" style={{padding:"-4em", marginLeft:"3.4em",display: "inline"}}>Total Games Played</label>
            <label id ="plays"/>
            </form>
            </div>
            </>
        )
    }
}
export default Tally
