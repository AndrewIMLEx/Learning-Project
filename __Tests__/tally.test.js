import React from 'react';
import Tally from '../src/Homepage/Tally';
import renderer from 'react-test-renderer';
describe("Tally page",() =>{
    it('renders the tally page', () => {
      const tree = renderer.create(<Tally />).toJSON();
      expect(tree).toMatchSnapshot();
    })
  })
