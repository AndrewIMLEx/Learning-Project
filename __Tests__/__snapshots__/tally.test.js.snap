// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tally page renders the rules page 1`] = `
<div
  className="tallycard"
>
  <h4
    style={
      {
        "marginLeft": "4em",
        "paddingBottom": "2em",
      }
    }
  >
    Game Tally
  </h4>
  <form
    id="LevelForm"
  >
    <span>
      <label
        htmlFor="won"
        id="won-label"
      >
        Games Won
      </label>
      <label
        id="Tally"
      />
    </span>
      
    <span>
      <label
        htmlFor="lost"
        id="won-label"
      >
        Games Lost
      </label>
       
      <label
        id="Tally"
      />
    </span>
  </form>
  <form
    id="Gamesplayed"
  >
    <label
      htmlFor="gamesplayed"
      id="played"
      style={
        {
          "display": "inline",
          "marginLeft": "4.5em",
          "padding": "-4em",
        }
      }
    >
      Total Games Played
    </label>
    <label
      id="plays"
    />
  </form>
</div>
`;
